runtime/src/main/java/org/corfudb/protocols/wireprotocol/failuredetector/NodeRank.java

```diff
@@ -49,13 +50,13 @@ public boolean is(String endpoint) {
     }
 
     public NodeRank(ByteBuf buf){
-        endpoint = ICorfuPayload.fromBuffer(buf, String.class);
-        numConnections = ICorfuPayload.fromBuffer(buf, Integer.class);
+        endpoint = CorfuProtocolCommon.fromBuffer(buf, String.class);
+        numConnections = CorfuProtocolCommon.fromBuffer(buf, Integer.class);
     }
 
     @Override
     public void doSerialize(ByteBuf buf) {
-        ICorfuPayload.serialize(buf, endpoint);
-        ICorfuPayload.serialize(buf, numConnections);
+        CorfuProtocolCommon.serialize(buf, endpoint);
+        CorfuProtocolCommon.serialize(buf, numConnections);
     }
```

Comment:
I think these can be removed since this class no longer needs to implement `ICorfuPayload`.

Reply:
Done.